generator-scrivito/app/index.js

```diff
@@ -6,6 +6,12 @@ module.exports = class extends Generator {
     this.log("Welcome to the Scrivito Obj/Widget generator");
   }
 
+  usage() {
+    return `yo scrivito
+  yo scrivito:obj [<name>]
```

Comment:
Where does the `[<name>]` notation come from? For me it was misleading because I tried the literal.
```zsh
yo scrivito:obj [FaqPage]
```

* [ ] FYI Maybe rewrite this for the common case `yo scrivito:obj <name>` or use another standard form (man style): `yo scrivito:obj [name ...]`.

However, if the `[<>]` notation is widely accepted in the yeoman world, fine with me.

Reply:
That is the default notation of `yo`. E.g. `yo scrivito:obj --help` will show:

```Usage:
  yo scrivito:obj [<name>] [options]

Options:
  -h,   --help           # Print the generator's options and usage
        --skip-cache     # Do not remember prompt answers             Default: false
        --skip-install   # Do not automatically install dependencies  Default: false
        --force-install  # Fail on install dependencies error         Default: false

Arguments:
  name  # The name of the new Obj class (e.g. FaqPage)  Type: String  Required: false
```

Reply:
I see. If the options is formatted as `[options]` the names should be formatted as `[names]`. We could also be a bit clearer with sth like `[obj_class_names]`. Not sure about the casing, but it's only an optional idea.